### PR TITLE
Update no-changes.tsx

### PR DESCRIPTION
**Pull request by Rexogamer**
## Overview

**Closes #{issue number}**

## Description

-

## Release notes

Notes:

### CODE REVIEW

**Comment by Rexogamer** [comment]
Pinging @tierninho

**Comment by Rexogamer** [comment]
This is for #8007 btw

File: app/src/ui/changes/no-changes.tsx [hunk 1]
```diff
@@ -582,7 +582,7 @@ export class NoChanges extends React.Component<
       <>
         You have{' '}
         {aheadBehind.ahead === 1 ? 'one local commit' : 'local commits'} waiting
-        to be pushed to {isGitHub ? 'GitHub' : 'the remote'}
+        to be pushed to {isGitHub ? 'GitHub' : 'the remote'}.
```
Comment (tierninho) [inline app/src/ui/changes/no-changes.tsx]: @Rexogamer Would you be so kind to make this change (above) under the branch in https://github.com/desktop/desktop/pull/8007 ? as a PR is already open for rewording the uncommitted changes text. 

Happy to give you the credit for it all 😄

Comment (Rexogamer) [reply app/src/ui/changes/no-changes.tsx]: I don’t know if I can edit stuff in that PR

Comment (Rexogamer) [reply app/src/ui/changes/no-changes.tsx]: @tierninho

Comment (Rexogamer) [reply app/src/ui/changes/no-changes.tsx]: This PR directs to the branch that PR merges, so the changes in this will be in that if merged

Comment (tierninho) [reply app/src/ui/changes/no-changes.tsx]: hmm, not sure. I went ahead and made the change.

**Comment by tierninho** [comment]
closing in favor of #8007

**Comment by Rexogamer** [comment]
I saw you brought over that period to the main PR; note that I also changed the text mentioned there

**Pull request closed without merging**